doc/classes/Geometry2D.xml

```diff
@@ -266,5 +267,11 @@
 		<constant name="END_ROUND" value="4" enum="PolyEndType">
 			Endpoints are rounded off and extended by [code]delta[/code] units.
 		</constant>
+		<constant name="RIGHT_HANDED" value="0" enum="PolyHandedness">
+			Vertices are in a right-handed coordinate system (the y-axis is positive upwards).
```

Comment:
It's actually the polarity of both axes I think. If both polarities are the same, it is right-handed, else left-handed. See the diagrams I posted on the previous PR.

Reply:
Listed all cases in 8e679d73b0be97c74c0c482c0e78fa8fbcd408b1.